go/x509/root_darwin.go

```diff
@@ -13,14 +13,14 @@ package x509
 #include <CoreFoundation/CoreFoundation.h>
 #include <Security/Security.h>
 
-// FetchPEMRoots fetches the system's list of trusted X.509 root certificates.
+// _FetchPEMRoots fetches the system's list of trusted X.509 root certificates.
```

Comment:
@pphaneuf @AlCutter I've updated it to have the suffix `CTX509`, to distinguish it from other applications that might use this CGO to fetch Darwin system roots.